h2o-py/h2o/explanation/_explain.py

```diff
@@ -472,7 +471,7 @@ def _get_algorithm(model,  treat_xrt_as_algorithm=False):
     return model.algo
 
 
-def _first_of_family(models, all_stackedensembles=True):
+def _first_of_family(models, all_stackedensembles=False):
```

Comment:
Yes, since all other models had only one "best of family" member, stacked ensembles were handled differently so we show all of them which is not preferred now that there's more of them.